ghost/core/core/server/models/index.js

```diff
@@ -1,35 +1,87 @@
-/**
- * Dependencies
- */
-
-const _ = require('lodash');
-const debug = require('@tryghost/debug')('models');
-const glob = require('glob');
+/* eslint-disable max-lines */
 
 // enable event listeners
 require('./base/listeners');
 
 /**
  * Expose all models
  */
-exports = module.exports;
+module.exports = {
+    // `base` file does not export a Base model
+    Base: require('./base'),
 
-function init() {
-    const baseNow = Date.now();
-    exports.Base = require('./base');
-    debug(`${Date.now() - baseNow}ms - Base.js require`);
-
-    let modelsFiles = glob.sync('!(index).js', {cwd: __dirname});
-    modelsFiles.forEach((model) => {
```

Comment:
Does it make sense to add this to a test? e.g. that all `ghost/core/core/server/models/*.js` are available/in the require cache?

Reply:
Good shout! I'll add that once I figure out this browser test failure 😆

Reply:
The rabbit gave a pretty good starting point 🐇🧙‍♂️ https://github.com/TryGhost/Ghost/pull/22358#discussion_r1991955011